tests/roles/swift_adoption/defaults/main.yaml

```diff
@@ -11,7 +11,7 @@ swift_patch: |
           replicas: 0
           networkAttachments:
           - storage
-          storageClass: local-storage
+          storageClass: {{ storage_class_name }}
```

Comment:
@cschwede Do you think we could remove the storageClass setting from here altogether? (Would it inherit from the OpenStackControlPlane CR setting?) The reason i didn't remove it yet is that it's actually explicitly mentioned in docs:

https://openstack-k8s-operators.github.io/data-plane-adoption/user/#adopting-the-object-storage-service_adopt-control-plane 

"Must match the RHOSO deployment storage class."

That makes me wonder whether we really need to repeat the setting here, or whether we can inherit from the storageClass setting on the control plane.